src/Lib/Server/Auth.hs

```diff
@@ -1,7 +1,8 @@
-{-# LANGUAGE DataKinds       #-}
-{-# LANGUAGE TemplateHaskell #-}
-{-# LANGUAGE TypeOperators   #-}
-
+{-# LANGUAGE DataKinds        #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE RecordWildCards  #-}
+{-# LANGUAGE TemplateHaskell  #-}
+{-# LANGUAGE TypeOperators    #-}
```

Comment:
Okay, instead of writing extensions in every module it's better to write `default-extensions` in `package.yaml` how it's done in `CDMP` and remove al language pragmas from top of the modules

* https://github.com/Holmusk/CDMP/blob/c9f1ba8513adf13720dbf5a73ed4b48afdd2c1aa/backend/package.yaml#L68-L81

You need to add exactly the same extensions as in CDMP.